README.md

```diff
@@ -1,5 +1,7 @@
 # GitHub Action for Azure CLI
 
+> [!WARNING]
+> The base image of Azure CLI's docker image is switched to Azure Linux since version `2.64.0`. The `az` commands will remain unaffected. To ensure a smooth transition, please migrate Alpine-specific commands to Azure Linux commands in azure/cli action. For more information: https://go.microsoft.com/fwlink/?linkid=2282203.

```

Comment:
```suggestion
> The base image of Azure CLI's docker image is switched to Azure Linux since version `2.64.0`. The `az` commands will remain unaffected. To ensure a smooth transition, please migrate Alpine-specific commands to Azure Linux commands in azure/cli action. For more information, see https://go.microsoft.com/fwlink/?linkid=2282203.

```